src/target/riscv/batch.h

```diff
@@ -55,8 +55,10 @@ void riscv_batch_free(struct riscv_batch *batch);
 /* Checks to see if this batch is full. */
 bool riscv_batch_full(struct riscv_batch *batch);
 
-/* Executes this scan batch. */
-int riscv_batch_run(struct riscv_batch *batch);
+/* Executes this scan batch. If resets_delays is true, after
+ * reset_delays_after_n_scans does not insert RTI cycles between scans. */
+int riscv_batch_run(struct riscv_batch *batch, bool resets_delays,
+		size_t reset_delays_after_n_scans);
```

Comment:
I believe the argument can be renamed to just `reset_delays_after`.

```suggestion
		size_t reset_delays_after);
```

Reply:
Addressed.